Kickstarter-iOS/Views/Storyboards/Settings.storyboard

```diff
@@ -681,20 +681,8 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="xml-QE-znh">
```

Comment:
Removed this manually added "header view" and instead created an actual table view header. This scene is now just a table view.